plugin/src/test/groovy/org/gradle/testretry/AbstractPluginFuncTest.groovy

```diff
@@ -132,6 +150,10 @@ abstract class AbstractPluginFuncTest extends Specification {
         return "acme.FlakyAssert.flakyAssert(\"${StringEscapeUtils.escapeJava(id)}\", $failures);"
     }
 
+    static String flakyAssertPassFailPass(String id = "id") {
+        return "acme.FlakyAssert.flakyAssertPassFailPass(\"${StringEscapeUtils.escapeJava(id)}\");"
```

Comment:
🤔 you could use `"""` or as string delimiters to avoid having to use escapes for `\"`